releasenotes/notes/qsphere-state-display-819cc1e3e61314a2.yaml

```diff
@@ -0,0 +1,19 @@
+---
+prelude: >
```

Comment:
Can you remove this section and integrate into the features section? It probably makes sense to make this a single feature note and describe each new kwarg as a list under it. 

The prelude section is for a high level overview of the entire release (whenever that happens) and it doesn't handle multiple elements well (it doesn't make bulleted lists for multiple entries and instead concatenates all prelude sections for a release. For an example of how the prelude is typically used see: https://github.com/Qiskit/qiskit-terra/blob/master/releasenotes/notes/0.14/0.14.0-release-19557dcd9d5af6e3.yaml#L2

Reply:
It was already duplicated within the features section, so I simply removed the prelude.